catroid/src/main/java/org/catrobat/catroid/visualplacement/VisualPlacementTouchListener.java

```diff
@@ -42,31 +59,46 @@ public boolean onTouch(ImageView imageView, MotionEvent event, CoordinateInterfa
 		if (event.getPointerId(0) == 0) {
 			float currentX = event.getRawX();
 			float currentY = event.getRawY();
+			long timeSinceTouchDown = event.getEventTime();
```

Comment:
Oh, thanks for correcting this. I misunderstood the function. OK, in that case, however, the name I suggested is not correct anymore, since it now again is only a timestamp. I thus propose to rename it here (and at all occurrences) to:
```suggestion
			long motionEventTime = event.getEventTime();
```